resources/test_data.json

```diff
@@ -63,12 +63,12 @@
         "commit_hash": "",
         "project_name": "jOOQ",
         "build_command": "mvn -V -B -e clean compile -Dcheckerframework.checkers=org.checkerframework.checker.signedness.SignednessChecker -Dcheckerframework.suppress=\"type.anno.before\"",
-        "root_dir": "src/main/java",
+        "root_dir": "jOOQ/src/main/java",
```

Comment:
These changes look like they are unrelated to the rest of this PR, and they should have been made separately.